packages/pages/src/components/image/image.test.tsx

```diff
@@ -0,0 +1,219 @@
+import React from "react";
+import {Image, getImageUUID, handleLayout} from "./image";
+import {ImageLayout} from "./types";
+import { render, screen } from '@testing-library/react';
+
+const imgWidth = 20;
+const imgHeight = 10;
+const imgUUID = "uuid";
+const width = 200;
+const height = 100;
+const aspectRatio = 1;
+const image = {
+  image: {
+    width: imgWidth,
+    height: imgHeight,
+    url: `https://a.mktgcdn.com/p/${imgUUID}/2x1.jpg`,
+  },
+};
+
+/**
+ * @jest-environment node
+ */
+
+describe("Image", () => {
+  it(`properly logs warning when layout is not ${ImageLayout.FIXED} and width or height is provided`, async () => {
+    jest.clearAllMocks();
```

Comment:
Shouldn't be needed at the beginning if your other tests properly clear at the end